docs/blueprints/EP028.rst

```diff
@@ -0,0 +1,153 @@
+
+
+Integration
+===========
+
+The following is the proposed solution with ES, Kibana and `Filebeat <https://www.elastic.co/beats/filebeat>`_ service. Filebeat is a lightweight shipper for logs, that's also supports `several input sources <https://www.elastic.co/guide/en/beats/filebeat/current/configuration-filebeat-options.html#filebeat-input-types>`_ (files, TCP, UDP, and many other protocols or services), with backpressure capabilities when sending data to ES. In this integration, the input type ``log`` using a file handler will be used to integrate reliably and seamlessly with logging.ini:
+
+
+.. code-block:: none
+
+    +---------------------+        +---------------------+  Submit log queries
+    |                     |        |                     |
+    |                     |        |                     |
+    |Elasticsearch service|<-------+   Kibana service    |<---------- Network Operators/Developers
+    |                     |        |                     |
+    |                     |        |                     |
+    +-----------^---------+        +---------------------+
+                |
+                | send
+    +-----------+---------+
+    |                     |
+    |                     |
+    | Filebeat service    |
+    |                     |
+    |                     |
+    +----------^----------+
+               |
+               | input source: log file
+               |
+    +----------+----------+
+    |                     |
+    |                     |
+    | kytos.log (file)    |
+    |                     |
+    |                     |
+    +----------^----------+
+               |
+               | write
+               |
+    +----------+----------+
+    |                     |
+    |                     |
+    |                     |
+    | RotatingFileHandler |
+    |                     |
+    | logging.ini         |
+    |                     |
+    |                     |
+    |                     |
+    |                     |
+    +---------------------+
+
+
+As represented in this diagram, as long as ``logging.ini`` has a handler that's writing to a file, Filebeat can read from it as an input, so it's up to network operators to configure it accordingly and also make sure to use the same formatter and dissect processors to parse and structure the fields of the log messages. For instance, this ``logging.ini`` below is based on `amlight/kytos-docker logging.ini <https://github.com/amlight/kytos-docker/blob/master/logging.ini>`_:
+
+.. code-block:: none
+
+    [formatters]
+    keys: file
+
+    [loggers]
+    keys: root
+
+    [formatter_file]
+    format: %(asctime)s - %(levelname)s [%(name)s] [%(filename)s:%(lineno)d:%(funcName)s] (%(threadName)s) %(message)s
+
+    [handler_file]
+    class: handlers.RotatingFileHandler
```

Comment:
@italovalcy, let me know what you think of having  `handlers.RotatingFileHandler` instead of `FileHandler` or if you have other suggestion, I think if we can leverage the rotating capabilities it can make a bit easier when we have to deal with rotations that we've captured in some discussion in the past. If we'd go for the rotating file handler here, we'd also need to get a ballpark figure of the the max size of the file and number of copies, in this initial draft to get the discussion started, it's using 10MB with 5 copies, and always appending to `/var/log/kytos/kytos.log`.

Reply:
Right @italovalcy, I've captured this discussion on this issue https://github.com/amlight/kytos-docker/issues/16, just so we have this issue on our radar.

Reply:
Much appreciated your review, @italovalcy